.github/workflows/release-scala-client.yaml

```diff
@@ -0,0 +1,13 @@
+name: Release Scala client
```

Comment:
Just by convention, this file is called `release.yml` in [all the other repos](https://github.com/guardian/gha-scala-library-release-workflow/issues/20) that have `gha-scala-library-release-workflow` - would it be ok to rename this file to match?

Reply:
The trouble is that this repo has a node client and a Scala client. I didn't want the workflow to give the impression it would do both. Let me know if you feel super strongly, but I don't think hogging the word "release" in the workflow namespace is always a good idea!

Reply:
That makes sense - what we've actually done with other libraries that publish both NPM & Scala packages is that we've tied them together in a single release workflow (at which point, it's kind of ok to just call it `release.yml` again). We allow the NPM release to utilise the Scala libraries version number, so that their releases are synchronised and it's easy to see whether the NPM and Scala libraries are from the same point in time.

We've done this for:

* https://github.com/guardian/content-api-models/pull/232
* https://github.com/guardian/content-entity/pull/32
* https://github.com/guardian/content-atom/pull/166

![image](https://github.com/guardian/content-api-models/assets/52038/9e2737c9-f75a-4b2f-9a10-824c4b1afc9b)

These libraries where largely completely defined by their thrift definitions, so if one library wanted to change, the other did too. This might not be the case with `anghammarad`, so it might not apply here, but it is quite nice to just have one button to press.

Reply:
Ok makes sense! I'll rename it and we can consider unifying these libraries in the future.